grails-app/views/product/_productSynonyms.gsp

```diff
@@ -45,11 +60,35 @@
 
                                       url="[controller: 'product', action:'addSynonymToProduct']">
                             <input name="id" type="hidden" value="${productInstance?.id}" />
-                            <input id="synonym" type="text" name="synonym" value="" size="80" class="medium text"/>
-                            <button  class="button">
-                                <img src="${createLinkTo(dir:'images/icons/silk', file:'add.png')}" />&nbsp;
-                                ${warehouse.message(code:'default.button.add.label')}
-                            </button>
+                            <div class="d-flex w-75 gap-8">
```

Comment:
It would be a good idea to already add translations for the input placeholders 
_Select a locale_
_Select a classification_
_Type the synonym here_

Reply:
yeah, also forgot to add it immediately, then you obviously forget to go back to such minor things :rofl:  thanks